### DISCUSSION TITLE
[DEV] ACore server LXD container image

### DISCUSSION THREAD

**Discussion by nolt** [azerothcore/azerothcore-wotlk#3884] [Tools]
Since some time I keep up2date ready2go LXD container with ACore based on Ubuntu20.04 and MySQL. I try to update image every week or two (compile+install+mysql updates). 
It's clean compiled AzerothCore from sources, it's not include any additional modules or custom SQL updates.

More you can find here:
https://github.com/nolt/acore-lxd-image

Since GIT not allow files larger than 2GB I'm uploading imgage to mega.

**Comment by nolt** [comment]
Please upgrade your LXD img ASAP.

New image contains:
- AzerothCore v3.0.0 commit 520c4c64269fdfa5468c54874247ead2bdece2dc
- AzerothCore configuration files updated
- game data files updated to v9
- Ubuntu kernel 5.8.0-40
- all system packages upgraded and updated

Have fun and support AzerothCore.

**Comment by nolt** [comment]
Image updated with recent ACE fixes, *.conf files has been merged. The image is based on ACore commit 6a7155e

**Comment by nolt** [comment]
Image update to ACore commit https://github.com/azerothcore/azerothcore-wotlk/commit/ae7b452fdfc49649b6e37ee23a8917d996d7d56c
- client data updated to v10
- merged *.conf files

**Comment by nolt** [comment]
Urgent update with client data update:
- fixed client data v10
- based on rev https://github.com/azerothcore/azerothcore-wotlk/commit/c05fa1a0a915ab90e8d7731ad0f8f3b2ff2e7731
- two images to choose MariaDB or Oracle MySQL

**Comment by nolt** [comment]
Since today I'll stop providing full images of LXD containers, there might be from time to time updates of full images but from now on LXD container will be created from scratch on host machine using script.

**Comment by nolt** [comment]
Time has come to update full images branch. This was needed as client data has been changed to v11.

**Reply by FrancescoBorzi** [reply]
thanks @nolt !
